app/src/main/kotlin/com/wire/android/ui/home/HomeScreen.kt

```diff
@@ -31,6 +36,9 @@ import com.wire.android.ui.common.topappbar.search.AppTopBarWithSearchBar
 fun HomeScreen(startScreen: String?, viewModel: HomeViewModel) {
     viewModel.checkRequirements()
     val homeState = rememberHomeState()
+    val coroutineScope = rememberCoroutineScope()
```

Comment:
I think coroutinescope is inside rememberHomeState if not could we move it there ?